test/InputProcessor.spec.ts

```diff
@@ -0,0 +1,85 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {parse} from '@asyncapi/parser';
+import { InputProcessor } from '../src/InputProcessor';
+import { AsyncAPIInputProcessor } from '../src/processors/AsyncAPIInputProcessor';
+import { JsonSchemaInputProcessor } from '../src/processors/JsonSchemaInputProcessor';
+describe('InputProcessor', function() {
+    let mainProcessor : InputProcessor;
+    beforeAll(() => {
+        mainProcessor = new InputProcessor();
+        const asyncAPI = new AsyncAPIInputProcessor();
+        mainProcessor.addProcessor("asyncapi", asyncAPI); 
```

Comment:
We should have included  `asyncapi` and `json-schema`  processors  in processors map as default.

Reply:
Alright, thought they would be added afterwards  👍